### DISCUSSION TITLE
search packages criteria

### DISCUSSION THREAD

**Discussion by ghost** [npm/feedback#892] [Registry] [answered]
Hello,
I couldn't find in the documentation an explanation on these things:
1. How is the popularity criterion determined? I would like to understand what exactly does it stand for. Would also like to know how are the "quality" and "maintenance" numbers are determined.
- My goal is to get approximately the "most wanted" packages.
3. What does "boost-exact" qualifier means?

Thank you!

**Comment by monishcm** [comment, marked as answer]
Hi, there is an ongoing [discussion](https://github.com/npm/feedback/discussions/66) on the related topic. Request you to follow that conversation about PQM scoring and some of the feedback around it that we are working on.
